CWWKE0701E: bundle com.ibm.ws.ssl ... The activate method has thrown an exception java.lang.ExceptionInInitializerError

**Describe the bug**  
 SSLcomponent can not be enabled on liberty 24.0.0.3 with IBM Java 8.0.8.20 with error CWWKE0701E:

{"type":"liberty_message","host":"4126b3e9a045","ibm_userDir":"/opt/ibm/wlp/usr/","ibm_serverName":"defaultServer","message":"CWWKE0701E: bundle com.ibm.ws.ssl:1.6.87.cl240320240311-1901 (30)[com.ibm.ws.ssl.internal.SSLComponent(219)] : The activate method has thrown an exception java.lang.ExceptionInInitializerError
at java.lang.J9VMInternals.ensureError(J9VMInternals.java:157)
at java.lang.J9VMInternals.recordInitializationFailure(J9VMInternals.java:146)
at com.ibm.ws.ssl.config.SSLConfigManager.getInstance(SSLConfigManager.java:112)
at com.ibm.ws.ssl.internal.SSLComponent.processConfig(SSLComponent.java:337)
at com.ibm.ws.ssl.internal.SSLComponent.activate(SSLComponent.java:126)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:90)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:55)
at java.lang.reflect.Method.invoke(Method.java:508)
at org.apache.felix.scr.impl.inject.methods.BaseMethod.invokeMethod(BaseMethod.java:257)
at org.apache.felix.scr.impl.inject.methods.BaseMethod.access$500(BaseMethod.java:41)
at org.apache.felix.scr.impl.inject.methods.BaseMethod$Resolved.invoke(BaseMethod.java:701)
at org.apache.felix.scr.impl.inject.methods.BaseMethod.invoke(BaseMethod.java:544)
at org.apache.felix.scr.impl.inject.methods.ActivateMethod.invoke(ActivateMethod.java:317)
at org.apache.felix.scr.impl.inject.methods.ActivateMethod.invoke(ActivateMethod.java:307)
at org.apache.felix.scr.impl.manager.SingleComponentManager.createImplementationObject(SingleComponentManager.java:354)
at org.apache.felix.scr.impl.manager.SingleComponentManager.createComponent(SingleComponentManager.java:115)
at org.apache.felix.scr.impl.manager.SingleComponentManager.getService(SingleComponentManager.java:1002)
at org.apache.felix.scr.impl.manager.SingleComponentManager.getServiceInternal(SingleComponentManager.java:975)
at org.apache.felix.scr.impl.manager.AbstractComponentManager.activateInternal(AbstractComponentManager.java:776)
at org.apache.felix.scr.impl.manager.AbstractComponentManager.enableInternal(AbstractComponentManager.java:674)
at org.apache.felix.scr.impl.manager.AbstractComponentManager.enable(AbstractComponentManager.java:437)
at org.apache.felix.scr.impl.manager.ConfigurableComponentHolder.enableComponents(ConfigurableComponentHolder.java:671)
at org.apache.felix.scr.impl.BundleComponentActivator.initialEnable(BundleComponentActivator.java:310)
at org.apache.felix.scr.impl.Activator.loadComponents(Activator.java:613)
at org.apache.felix.scr.impl.Activator.access$200(Activator.java:75)
at org.apache.felix.scr.impl.Activator$ScrExtension.start(Activator.java:480)
at org.apache.felix.scr.impl.AbstractExtender.createExtension(AbstractExtender.java:196)
at org.apache.felix.scr.impl.AbstractExtender.modifiedBundle(AbstractExtender.java:169)
at org.apache.felix.scr.impl.AbstractExtender.modifiedBundle(AbstractExtender.java:49)
at org.osgi.util.tracker.BundleTracker$Tracked.customizerModified(BundleTracker.java:490)
at org.osgi.util.tracker.BundleTracker$Tracked.customizerModified(BundleTracker.java:1)
at org.osgi.util.tracker.AbstractTracked.track(AbstractTracked.java:234)
at org.osgi.util.tracker.BundleTracker$Tracked.bundleChanged(BundleTracker.java:452)
at org.eclipse.osgi.internal.framework.BundleContextImpl.dispatchEvent(BundleContextImpl.java:949)
at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:234)
at org.eclipse.osgi.framework.eventmgr.ListenerQueue.dispatchEventSynchronous(ListenerQueue.java:151)
at org.eclipse.osgi.internal.framework.EquinoxEventPublisher.publishBundleEventPrivileged(EquinoxEventPublisher.java:229)
at org.eclipse.osgi.internal.framework.EquinoxEventPublisher.publishBundleEvent(EquinoxEventPublisher.java:138)
at org.eclipse.osgi.internal.framework.EquinoxEventPublisher.publishBundleEvent(EquinoxEventPublisher.java:130)
at org.eclipse.osgi.internal.framework.EquinoxContainerAdaptor.publishModuleEvent(EquinoxContainerAdaptor.java:217)
at org.eclipse.osgi.container.Module.publishEvent(Module.java:499)
at org.eclipse.osgi.container.Module.start(Module.java:486)
at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel$2.run(ModuleContainer.java:1852)
at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel$1.execute(ModuleContainer.java:1840)
at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1845)
at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1789)
at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1750)
at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1672)
at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1)
at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:234)
at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:345)\nCaused by: java.lang.NullPointerException
at com.ibm.ws.ssl.provider.IBMJSSEProvider.(IBMJSSEProvider.java:66)
at com.ibm.ws.ssl.JSSEProviderFactory.getInstance(JSSEProviderFactory.java:138)
at com.ibm.ws.ssl.JSSEProviderFactory.getInstance(JSSEProviderFactory.java:66)
at com.ibm.ws.ssl.config.SSLConfigManager.(SSLConfigManager.java:103)
at com.ibm.ws.ssl.config.SSLConfigManager.(SSLConfigManager.java:66)
at com.ibm.ws.ssl.config.SSLConfigManager$Singleton.(SSLConfigManager.java:70)

**Steps to Reproduce**  
Move up from previous Liberty version to Liberty version 24.0.0.3
**Expected behavior**  
Should be able to move up from previous Liberty version to Liberty version 24.0.0.3

**Diagnostic information:**  
 - OpenLiberty Version: 24.0.0.3 
 - Affected feature(s) [ssk-1.0 and transportSecurity-1.0  ]
 - Java Version: all versions

--- a/dev/com.ibm.ws.crypto.ltpakeyutil/src/com/ibm/ws/crypto/ltpakeyutil/LTPAKeyUtil.java
+++ b/dev/com.ibm.ws.crypto.ltpakeyutil/src/com/ibm/ws/crypto/ltpakeyutil/LTPAKeyUtil.java
@@ -110,7 +110,7 @@ public final class LTPAKeyUtil {
 				}
 			});
 			ibmJCEPlusFIPSProviderChecked = true;
-			if (ibmjceplusfipsprovider == "IBMJCEPlusFIPS" && isRunningBetaMode()) {
+			if (isRunningBetaMode() && "IBMJCEPlusFIPS".equalsIgnoreCase(ibmjceplusfipsprovider)) {
 				ibmJCEPlusFIPSAvailable = true;
 				return ibmJCEPlusFIPSAvailable;
 			} else {

--- a/dev/com.ibm.ws.ssl/src/com/ibm/ws/ssl/provider/IBMJSSEProvider.java
+++ b/dev/com.ibm.ws.ssl/src/com/ibm/ws/ssl/provider/IBMJSSEProvider.java
@@ -62,8 +62,7 @@ public class IBMJSSEProvider extends AbstractJSSEProvider implements JSSEProvide
         if (tc.isDebugEnabled()) {
             Tr.debug(tc, "provider: " + ibmjceplusfipsprovider);
         }
-
-        if (fipsON != null && fipsON.equalsIgnoreCase("true") && ibmjceplusfipsprovider.equals("IBMJCEPlusFIPS") && isRunningBetaMode()) {
+        if (isRunningBetaMode() && "true".equalsIgnoreCase(fipsON) && "IBMJCEPlusFIPS".equalsIgnoreCase(ibmjceplusfipsprovider)) {
             if (tc.isDebugEnabled()) {
                 Tr.debug(tc, "fips is enabled and using IBMJCEPlusFIPS provider");
                 Tr.debug(tc, "key manager factory alg: " + JSSEProviderFactory.getKeyManagerFactoryAlgorithm());

